.github/workflows/drivers_ci.yml

```diff
@@ -148,18 +141,11 @@ jobs:
           cd bpftool
           git submodule update --init
           cd src && sudo make install
-          
-      - name: Install kernel headers (workaround) and gcc
+
+      - name: Install kernel headers (actuated)
+        uses: self-actuated/get-kernel-sources@master
         if: matrix.arch == 'arm64'
-        run: |
-          sudo apt install -y --no-install-recommends gcc g++
```

Comment:
We don't need to explicitly install `gcc` and `g++` since they are already installed as deps in previous steps:
```
apt install -y --no-install-recommends ca-certificates cmake build-essential clang-14 llvm-14 git pkg-config autoconf automake libtool libelf-dev libcap-dev

...

The following NEW packages will be installed:
  autoconf automake autotools-dev binutils binutils-common binutils-x86-64-linux-gnu build-essential bzip2 ca-certificates clang-14 cmake cmake-data cpp cpp-11 dh-elpa-helper dpkg-dev emacsen-common file g++
  g++-11 gcc gcc-11 gcc-11-base git git-man lib32gcc-s1 lib32stdc++6 libarchive13 libasan6 libatomic1 libbinutils libbrotli1 libbsd0 libc-dev-bin libc6-dev libc6-i386 libcap-dev libcc1-0
  libclang-common-14-dev libclang-cpp14 libclang1-14 libcrypt-dev libctf-nobfd0 libctf0 libcurl3-gnutls libcurl4 libdpkg-perl libedit2 libelf-dev libelf1 liberror-perl libexpat1 libgc1 libgcc-11-dev
  libgdbm-compat4 libgdbm6 libglib2.0-0 libgomp1 libicu70 libisl23 libitm1 libjsoncpp25 libldap-2.5-0 libllvm14 liblsan0 libmagic-mgc libmagic1 libmd0 libmpc3 libmpfr6 libnghttp2-14 libnsl-dev libobjc-11-dev
  libobjc4 libperl5.34 libpfm4 libpsl5 libquadmath0 librhash0 librtmp1 libsasl2-2 libsasl2-modules-db libsigsegv2 libssh-4 libstdc++-11-dev libtirpc-dev libtool libtsan0 libubsan1 libuv1 libxml2
  linux-libc-dev llvm-14 llvm-14-linker-tools llvm-14-runtime lto-disabled-list m4 make openssl patch perl perl-modules-5.34 pkg-config rpcsvc-proto xz-utils zlib1g-dev
```